.vscode/settings.json

```diff
@@ -19,7 +19,7 @@
   "typescript.referencesCodeLens.enabled": true,
   "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
This happened because of a VSCode change shift from 1.84 to 1.85, when boolean values for properties was deprecated. This change just shipped this month ( https://stackoverflow.com/a/77637765 ).